NVIDIA/deepops
# Allow user to override Slurm configuration with their own files

Slurm configuration is complex, and oftentimes needs a lot of manual tuning! Our templated configurations capture a lot of best practices, but many sites will still need to tweak these for their own needs.

This PR allows the user to specify their own configuration files via Ansible variables, so that they can add their custom configuration separately rather than editing the DeepOps role files.

## Test plan

1. Create your own static slurm.conf
1. Set `slurm_conf_template` to the absolute path of your file
1. Run the `slurm-cluster.yml` playbook and verify that you have the correct config file!
